doc/sphinx/source/drivers_doc.rst

```diff
@@ -18,6 +18,13 @@ ANALOG TO DIGITAL CONVERTERS
 
    drivers/adm1177
 
+DIGITAL INPUT/OUTPUT
+==========================
+.. toctree::
+   :maxdepth: 1
```

Comment:
after the merge of #1955, you need only `drivers/max22190` to add, since the ToC is already created. (therefore the newly appeared conflict.

Reply:
I thought about this when writing the code. Thinking of waiting for #1955 to be merged first, and then only add drivers/max22190 line in this file (since ToC will already be created).

Reply:
I merged it so you can resolve already the conflict :)

Reply:
On it.